README.md

```diff
@@ -118,6 +119,32 @@ Additional notes for non-coders
    the same code spot-check process a new contributor needs to undergo at the
    time they add the code.
 
+Backdrop Bug Squad
+------------------
+
+The Backdrop Bug Squad consistes of members of the backdrop contrib group who
+will help other contributors stay on top of minor bug fixes and UI improvements.
+
+Responsibilities include:
+- Communicating with existing maintainers.
+- Reviewing Pull Requests for Squad-Merge eligibility.
```

Comment:
Defining "Squad-Merge" or avoiding this term might be good. I'm not sure what this means. I also had difficulty incorrectly reading this as "Squash-Merge".

Reply:
removed.